rust_src/src/frames.rs

```diff
@@ -672,4 +680,12 @@ pub fn frame_face_alist(frame: LispFrameLiveOrSelected) -> LispObject {
     frame_ref.face_alist
 }
 
+/// Return the value of frame parameter PROP in frame FRAME.
+#[no_mangle]
+pub extern "C" fn get_frame_param(frame: LispFrameRef, prop: LispObject) -> LispObject {
```

Comment:
This is dangerous. `frame` might be NULL and never validate it. I see that the original C did not either so likely not a problem here. But something we should keep in mind while porting.

Reply:
I'd be glad to handle the NULL case here. Should we also return NULL if the frame is null?

Reply:
Let's keep an eye on it. I suspect as the port continues this function won't live long.